src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -1,28 +1,68 @@
 import React from 'react';
+import { useFormikContext } from 'formik';
+import { Spinner } from '@patternfly/react-core';
 import { HostSubnets, ClusterConfigurationValues } from '../../types/clusters';
 import { InputField, SelectField, SwitchField } from '../ui/formik';
-import { useFormikContext } from 'formik';
+import { Cluster } from '../../api/types';
+import StaticTextField from '../ui/StaticTextField';
+
+const getVipStaticValue = (vip?: string, vipDhcpAllocation?: boolean) => {
```

Comment:
nit: Since `StaticTextField` value is `ReactNode` now, this can be turned into a component. In addition, there is no need to memoize its result bellow.

Reply:
Done